networking/network_observability/network-observability-overview.adoc

```diff
@@ -6,14 +6,14 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The Network Observability uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored netflow information in the {product-title} console for further insight and troubleshooting.
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The Network Observability uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored network flow information in the {product-title} console for further insight and troubleshooting.
 
 [id="dependency-network-observability"]
 == Dependency of Network Observability Operator
 
 The Network Observability Operator requires the following Operators:
 
-* Loki: You must install Loki. Loki is the backend that is used to store all collected flows. It is recommended to install Loki by installing the Red Hat Loki Operator for the installation of Network Observability Operator.
+* Loki: You must install Loki. Loki is the backend, persistent log store for the network flow data. It is recommended to install Loki by installing the Red Hat Loki Operator for the installation of Network Observability Operator.
```

Comment:
https://github.com/openshift/openshift-docs/issues/54810
As for linking to logging with Loki, I'm exploring the option to use conditional text to consolidate use of a single file where Loki deployment is documented, so I did not include this link.

Reply:
Loki is not a log store.  Loki is the software or server that persists the data to a log store.  The original text is technically correct.

Reply:
I'll revert these updates.